src/modals/select-baker-modal/select-baker-modal.tsx

```diff
@@ -22,13 +28,36 @@ import { isString } from '../../utils/is-string';
 import { SelectBakerItem } from './select-baker-item/select-baker-item';
 import { useSelectBakerModalStyles } from './select-baker-modal.styles';
 
+// TODO: replace with translations
+const bakersSortFieldsLabels: Record<BakersSortFieldEnum, string> = {
+  [BakersSortFieldEnum.Fee]: 'Fee',
+  [BakersSortFieldEnum.Rank]: 'Rank',
+  [BakersSortFieldEnum.Space]: 'Space'
+};
+const bakersSortFieldsOptions = [BakersSortFieldEnum.Space, BakersSortFieldEnum.Fee, BakersSortFieldEnum.Rank];
+
+type BakerListItemProps = {
+  item: BakerInterface;
+  selected: boolean;
+  onPress: (item: BakerInterface) => void;
```

Comment:
use `EventFn` type   
+ you could just reuse `SelectBakerItem` `Props` here (they are the same)